Fix: ui testing framework - get fresh token date logic

patthiel opened the pull request:
* Use moment js to compare exact dates when checking if a token is expired. (dateModified > dateTwoHoursinFuture)

* Previously it was just comparing the hours (16 vs. 14) wouldn't get a new token, even if it was 16:50 vs. 14:01.

rjstires commented on e2e/utils/common.js:
```diff
@@ -27,8 +28,8 @@ export const login = (username, password) => {
-    const expirationDate = new Date().getHours() + 2 // Current Time + 2 hours;
-    const getNewToken = tokenExists ? new Date(statSync(tokenPath).mtime).getHours() > expirationDate: true;
+    const expirationDate = moment().add('2', 'hours').format();
+    const getNewToken = tokenExists ? moment(new Date(statSync(tokenPath).mtime)).format() > expirationDate: true;
```
While I agree with @asauber, if we're going to enforce this is should be done by the tooling.

patthiel replied:
Fixed the formatting. There was also an issue here because i was setting the expiration date to two hours in the future from the current date, not from the date two hours after the token last modified date. 

I corrected this and made the code safer/more readable.

patthiel commented:
Added some changes that make it more clear and fixed a mistake. Also I switched all the module syntax over to commonJS and removed references to babel-register.
